### DISCUSSION TITLE
Breaking changes in v11 - CHANGELOG

### DISCUSSION THREAD

**Discussion by MaxDonchenko** [danger/danger-js#1284] [Q&A]
Hey there! I can't find migration steps for v11 in the changelog:
https://danger.systems/js/changelog.html

Could you pls clarify whether there are no breaking changes or if there are - where I can find them?

**Comment by orta** [comment]
Looks like it's just out of date: https://github.com/danger/danger-js/blob/main/CHANGELOG.md#1100---1102
